Tests/MongoSwiftSyncTests/SpecTestRunner/FailPoint.swift

```diff
@@ -70,7 +70,22 @@ internal struct FailPoint: Decodable {
     }
 
     public init(from decoder: Decoder) throws {
-        self.failPoint = try BSONDocument(from: decoder)
+        let container = try decoder.singleValueContainer()
```

Comment:
This is required due to the ordering of the command getting messed up during decoding from JSON.